src/recursion.erl

```diff
@@ -0,0 +1,31 @@
+-module(recursion).
+
+%%
+%% Example:
+%% Different functions to capitalize a string
+%%
+
+%% BAD: makes unnecessary use of manual recursion
+recurse(S) ->
+    lists:reverse(recurse(S, [])).
+
+recurse([], Acc) ->
```

Comment:
Hahaha I wish I could claim credit for intentionally writing a bug in this function, but admittedly I did not actually test the code since I was just trying to convey the gist of the idea, and didn't expect anyone to actually run it 😆 . I will fix it along with addressing your other comment above, and then do a rebase and push.